src/GafferArnold/ArnoldDisplacement.cpp

```diff
@@ -200,7 +205,10 @@ bool ArnoldDisplacement::acceptsInput( const Gaffer::Plug *plug, const Gaffer::P
 	{
 		if( const GafferScene::Shader *shader = runTimeCast<const GafferScene::Shader>( inputPlug->source<Plug>()->node() ) )
 		{
-			return runTimeCast<const ArnoldShader>( shader );
+			return
```

Comment:
I mean, it definitely helps clarity, in the same way that our "always use `{}` even for a one-line `if`" convention helps clarity.